src/lib.rs

```diff
@@ -51,7 +62,10 @@ mod util;
 pub use self::error::Error;
 #[cfg(any(unix, windows, libloading_docs))]
 pub use self::safe::{Library, Symbol};
+
+#[cfg(feature = "std")]
 use std::env::consts::{DLL_PREFIX, DLL_SUFFIX};
+#[cfg(feature = "std")]
 use std::ffi::{OsStr, OsString};
```

Comment:
Nit: these uses could be moved to inside the function and/or references to them replaced by fully-qualified paths in order to avoid these getting accidental non-local use in future development more obviously.

(Of course the CI would catch such instances, but it would be pretty late for it.)